overlord/hookstate/ctlcmd/system_mode_test.go

```diff
@@ -102,23 +102,25 @@ func (s *systemModeSuite) TestSystemMode(c *C) {
 		},
 	}
 
-	for _, test := range tests {
-		smi = &test.smi
-		smiErr = test.smiErr
+	for uid := range []int{0 /* root */, 1000 /* regular */} {
```

Comment:
Don't you need to do
```golang
for _, uid := ...
```
here? Otherwise I think uid will take the values `0` and `1`.

Reply:
you are correct, I didn't notice because the test passes anyway, I will adjust/fix, thanks